api/envoy/api/v2/route/route.proto

```diff
@@ -942,6 +942,9 @@ message RetryPolicy {
   // the base interval. The documentation for :ref:`config_http_filters_router_x-envoy-max-retries`
   // describes Envoy's back-off algorithm.
   RetryBackOff retry_back_off = 8;
+
+  // HTTP headers that trigger a retry if present in the response (the value doesn't matter).
+  repeated string retriable_headers = 9;
```

Comment:
Let's make clear that this field only comes into play if retriable headers are enabled (viai `retry_on` field or headers).

Please update the next free field comment at the top of RetryPolicy.

Reply:
Done.